test/extended/util/framework.go

```diff
@@ -1054,6 +1054,11 @@ var CheckPodIsReadyFn = func(pod kapiv1.Pod) bool {
 	return false
 }
 
+// CheckPodExists returns true if the pod exists or had existed
```

Comment:
this godoc is misleading...   someone might read it and assume they could just invoke CheckPodExists(pod).  In reality it always returns true and can't be used standalone, it really only makes sense in the context of passing it as a no-op func to WaitForPods().

Reply:
I had considered updating this to do a more robust check (ex: does the pod have non-empty `ObjectMeta`)

Reply:
just renaming it to CheckPodNoOp would make me happy.